akka-http-marshallers-scala/akka-http-spray-json/src/main/scala/akka/http/scaladsl/marshallers/sprayjson/SprayJsonSupport.scala

```diff
@@ -64,3 +66,75 @@ trait SprayJsonSupport {
     Marshaller.StringMarshaller.wrap(MediaTypes.`application/json`)(printer)
 }
 object SprayJsonSupport extends SprayJsonSupport
+
+/**
+ * Entity streaming support, independent of used Json parsing library etc.
```

Comment:
it it's independent of the Json parsing library, should it then be in the `sprayjson` package?